### DISCUSSION TITLE
Suggestion: add "Open prefix/suffix slot" as potential stat to base item price on

### DISCUSSION THREAD

**Discussion by ThirteenEra** [SnosMe/awakened-poe-trade#705] [Ideas]
Price of two items with exact stats, where one has open suffix (or prefix) and other one doesnt can vary significantly. With awakened POE trade you have to manually search trade for these filters if you have item with open affix slots, since awakened POE trade doesnt seem to have option to include that in evaluation.

Can we please have it as an option?

**Comment by SnosMe** [comment]
This is available for a long time under "hidden" toggle (obviously when you have open pref/suff). It's there because in most cases this will cause error due to query complexity and you will need to login in built-in browser, which I don't want force people to do.

**Reply by ThirteenEra** [reply]
Not on all items.

For example, here is screenshot of me checking an item with open affixes and option to include open affixes in pricing not being available. This item has 1 prefix and 3 suffixes, allowing for potential crafts like life etc, thus increasing its value significantly.

<img width="790" alt="affix" src="https://user-images.githubusercontent.com/111650549/185880176-462dd043-d21b-4669-aec5-160119f23ef2.png">

**Reply by SnosMe** [reply]
I'm only showing it when 5 prefixes are filed and 6th is open/crafted.
IRC because I haven't figured out the correct set of filters for 4+2.
